Simplenote/SPAppDelegate.m

```diff
@@ -263,30 +263,6 @@ - (void)markFirstLaunch
     [[Options shared] setFirstLaunch:YES];
 }
 
-- (void)createWelcomeNoteAfterDelay
-{
-    [self performSelector:@selector(createWelcomeNote) withObject:nil afterDelay:0.5];
-}
-
-- (void)createWelcomeNote
-{
-    NSString *welcomeKey = @"welcomeNote-iOS";
-    SPBucket *noteBucket = [_simperium bucketForName:@"Note"];
-    Note *welcomeNote = [noteBucket objectForKey:welcomeKey];
-    
-    if (welcomeNote) {
-        return;
-	}
-    
-    welcomeNote = [noteBucket insertNewObjectForKey:welcomeKey];
-    welcomeNote.modificationDate = [NSDate date];
-    welcomeNote.creationDate = [NSDate date];
-    welcomeNote.content = NSLocalizedString(@"welcomeNote-iOS", @"A welcome note for new iOS users");
```

Comment:
We may also wanna (manually) drop `welcomeNote-iOS` from the .strings files (since that particular string was manually put there I believe!)

Reply:
Hmm, I think it will be gone when release is created. We will see :-)